regression/utils.py

```diff
@@ -3,12 +3,191 @@
+    if 'batch_size' in kwargs:
+        dataset_params['batch_size'] = int(kwargs['batch_size'])
+    if 'shuffle' in kwargs:
+        dataset_params['shuffle'] = kwargs['shuffle'].lower().capitalize() == 'True'
+    if 'shuffle_algo' in kwargs:
+        dataset_params['shuffle_algo'] = kwargs['shuffle_algo']
+    if 'shuffle_seed' in kwargs:
+        dataset_params['shuffle_seed'] = int(kwargs['shuffle_seed'])
+    if 'shuffle_block_size' in kwargs:
+        dataset_params['shuffle_block_size'] = int(kwargs['shuffle_block_size'])
+    if 'sampling_method' in kwargs:
+        dataset_params['sampling_method'] = kwargs['sampling_method']
+    if 'proportion' in kwargs:
+        dataset_params['proportion'] = float(kwargs['proportion'])
+    if 'repeat' in kwargs:
+        dataset_params['repeat'] = float(kwargs['repeat'])
+    if 'choose' in kwargs:
+        dataset_params['choose'] = int(kwargs['choose'])
+    logger.debug(f'dataset_params: {dataset_params}')
+    return dataset_params
+
+
+def get_dataloader_params(kwargs: Dict[str, str]) -> Dict[str, Any]:
+    """Get the dataloader parameters from command-line arguments.
+
+    Args:
+        kwargs (Dict[str, str]): Command-line arguments.
+
+    Returns:
+        Dict[str, Any]: Dataloader parameters.
+    """
+    dataloader_params = {}
+    if 'num_workers' in kwargs:
+        dataloader_params['num_workers'] = int(kwargs['num_workers'])
+    if 'batch_size' in kwargs:
+        dataloader_params['batch_size'] = int(kwargs['batch_size'])
+    if 'pin_memory' in kwargs:
+        dataloader_params['pin_memory'] = kwargs['pin_memory'].lower().capitalize() == 'True'
+    if 'persistent_workers' in kwargs:
+        dataloader_params['persistent_workers'] = kwargs['persistent_workers'].lower().capitalize(
+        ) == 'True'
+    logger.debug(f'dataloader_params: {dataloader_params}')
+    return dataloader_params
+
+
+def get_writer_params(kwargs: Dict[str, str]) -> Dict[str, Any]:
+    """Get the writer parameters from command-line arguments.
+
+    Args:
+        kwargs (Dict[str, str]): Command-line arguments.
+
+    Returns:
+        Dict[str, Any]: Writer parameters.
+    """
+    writer_params = {}
+    if 'keep_local' in kwargs:
+        writer_params['keep_local'] = kwargs['keep_local'].lower().capitalize() == 'True'
+    if 'compression' in kwargs:
+        writer_params['compression'] = kwargs['compression']
+    if 'hashes' in kwargs:
+        writer_params['hashes'] = kwargs['hashes'].split(',')
+    if 'size_limit' in kwargs:
+        writer_params['size_limit'] = str(kwargs['size_limit'])
+    if 'progress_bar' in kwargs:
+        writer_params['progress_bar'] = kwargs['progress_bar'].lower().capitalize() == 'True'
+    if 'max_workers' in kwargs:
+        writer_params['max_workers'] = int(kwargs['max_workers'])
+    logger.debug(f'writer_params: {writer_params}')
+    return writer_params
 
 
 def get_local_remote_dir() -> str:
```

Comment:
Is this function still being used?

Reply:
Good catch. Will remove it if not being used.